component/main.jsonnet

```diff
@@ -1,12 +1,21 @@
 // main template for openshift4-version
+local com = import 'lib/commodore.libjsonnet';
 local kap = import 'lib/kapitan.libjsonnet';
 local kube = import 'lib/kube.libjsonnet';
 local inv = kap.inventory();
 // The hiera parameters for the component
 local params = inv.parameters.openshift4_version;
 
+local cluster_gt_411 =
+  params.openshiftVersion.Major == '4' &&
+  std.parseInt(params.openshiftVersion.Minor) >= 11;
+
 local clusterVersion = kube._Object('config.openshift.io/v1', 'ClusterVersion', 'version') {
-  spec: params.spec,
+  spec: {
+    [if cluster_gt_411 then 'capabilities']: {
+      baselineCapabilitySet: 'v4.11',
```

Comment:
Might be worth to have this exposed as a parameter.

Reply:
You can already override this via `parameters.openshift4_version.spec.capabilities.baselineCapabilitySet` the same as all the other available fields of `ClusterVersion`.